ci-operator/infra/openshift/release-controller/releases/release-ocp-4.2.json

```diff
@@ -37,6 +37,10 @@
       "optional":true,
       "prowJob":{"name":"canary-openshift-ocp-installer-e2e-gcp-serial-4.2"}
     },
+    "aws-console":{
+      "optional":true,
+      "prowJob":{"name":"release-openshift-ocp-installer-console-aws-4.2"}
```

Comment:
If this is brand new and we haven't seen it pass, we want a `canary-` prefix first while the job is proven.

Reply:
OK, updated.